solidity/random-beacon/contracts/libraries/BLS.sol

```diff
@@ -47,12 +47,25 @@ library BLS {
     ) public view returns (bool) {
         AltBn128.G1Point memory _signature = AltBn128.g1Unmarshal(signature);
 
+        return
+            _verify(
+                AltBn128.g2Unmarshal(publicKey),
+                AltBn128.g1Unmarshal(message),
+                _signature
+            );
+    }
+
+    function _verify(
```

Comment:
Do we need to underscore it? The compiler should resolve overloaded versions correctly I think.

Reply:
Some tests break if I don't underscore it: `test/BLS.test.ts:18:30 - error TS2339: Property 'verify' does not exist on type 'BLS'.`

Reply:
Ethers can not resolve the correct function, we'd need to provide a signature as a string, e.g. `randomBeacon["getGroup(bytes)"]`.  Let's leave the underscore, it's not worth our pain in tests.